lib/ElementHandle.js

```diff
@@ -103,6 +103,26 @@ class ElementHandle {
     const objectId = this._remoteObject.objectId;
     return this._client.send('DOM.setFileInputFiles', { objectId, files });
   }
+
+  /**
+   * @return {!Promise<Object>}
+   */
+  async boundingBox() {
+    const box = await this.evaluate(element => {
+      if (!element.ownerDocument.contains(element))
+        return null;
+      const rect = element.getBoundingClientRect();
```

Comment:
Thanks for adding a test!

> but I ended up using the code from document-offset for getting the absolute offset.

In case of nested iframes, this will return the bounding box inside the iframe's document, not the main frame document. (can we have a test for this too?)